packages/react-scripts/config/env.js

```diff
@@ -27,6 +27,8 @@ function getClientEnvironment(publicUrl) {
       'NODE_ENV': JSON.stringify(
         process.env.NODE_ENV || 'development'
       ),
+      // Allows the chalk library to work on the browser
+      'FORCE_COLOR': JSON.stringify(true),
```

Comment:
Can we just set `process.env.FORCE_COLOR` right before importing chalk and immediately delete it afterwards right on the client?